test/profiles.js

```diff
@@ -0,0 +1,104 @@
+import 'babel-polyfill'
+import { assert } from 'chai'
+import fs from 'fs'
+
+import Util from '../src/utils'
+import { Profiles, validate } from '../src/profiles'
+
+describe('Profiles', () => {
+  describe('#retrieve', () => {
+    it('returns `null` if profile ID doesn\'t exist', async () => {
+      const profiles = await new Profiles(true)
+        , retrieved = profiles.retrieve('inexistent-profile-id')
+
+      assert(retrieved === null, 'Value should be null')
+    })
+
+    it('returns remote profile by its ID', async () => {
+      const profiles = await new Profiles(true)
+        , baseProfile = fs.readFileSync('schemas/data-package.json', 'utf8')
+        , retrieved = profiles.retrieve('base')
+
+      assert.deepEqual(retrieved, JSON.parse(baseProfile))
+    })
+
+    it('returns local schema by its ID', async () => {
```

Comment:
It's MINOR but example above says `remote profile` and here `local schema`. We have a deal here with a little bit confusing staff like having JSON Schema and at the same time (JSON) Table Schema etc. So consistency in naming in our code is pretty important to do not get lost.